src/Logary/Targets_Core.fs

```diff
@@ -597,14 +597,16 @@ module Debugger =
             ack *<= () :> Job<_>
 
           RingBuffer.take requests ^=> function
```

Comment:
All of this target should be conditionally compiled if it doesn't exist on .Net Core. To be honest I never use it and I don't know if someone else it; it's just a 'hey - it's easy to create new targets'-target for me.